docs/pages/faq.mdx

```diff
@@ -123,6 +123,18 @@ Teleport provides security-critical support for the current and two previous rel
 See our [Upgrading](./setup/operations/upgrading.mdx) guide for more
 information.
 
+## Which operating systems are supported?
+
+Teleport runs on Linux, macOS, and Windows (client only).
```

Comment:
Does "client only" also apply to remote use of `tctl`?

Reply:
No, we only support `tsh.exe` on Windows for now. Reworded it a bit, what do you think?